pinot-common/src/main/java/org/apache/pinot/common/assignment/InstancePartitionsUtils.java

```diff
@@ -59,13 +59,13 @@ public static String getInstancePartitionsName(String tableName, String instance
   /**
    * Fetches the instance partitions from Helix property store if it exists, or computes it for backward-compatibility.
    */
-  public static InstancePartitions fetchOrComputeInstancePartitions(HelixManager helixManager, TableConfig tableConfig,
-      InstancePartitionsType instancePartitionsType) {
+  public static InstancePartitions fetchOrComputeInstancePartitions(HelixManager helixManager,
```

Comment:
nit: let's maybe keep this consistent with the old code
```suggestion
  public static InstancePartitions fetchOrComputeInstancePartitions(HelixManager helixManager, TableConfig tableConfig,
InstancePartitionsType instancePartitionsType) {
```